airflow/utils/log/file_task_handler.py

```diff
@@ -187,10 +187,14 @@ def _read(self, ti, try_number, metadata=None):
                 response.encoding = "utf-8"
 
                 if response.status_code == 403:
-                    log += "*** !!!! Please make sure that all your webservers and workers have" \
-                           " the same 'secret_key' configured in 'webserver' section !!!!!\n***"
-                    log += "*** See more at https://airflow.apache.org/docs/apache-airflow/" \
-                           "stable/configurations-ref.html#secret-key\n***"
+                    log += (
+                        "*** !!!! Please make sure that all your webservers and workers have"
```

Comment:
```suggestion
                        "*** !!!! Please make sure that all your Airflow components (e.g. schedulers, webservers and workers) have"
```

Could also be the scheduler when using a local executor 🤷‍♂️

Reply:
Right. The If above is only for K8S executor. I always associated LocalExecutor with running everything on one machine and reading logs from filesystem, but indeed webserver might run separately. I am not sure though if Log server actually runs for Scheduler in this case ?

Reply:
Ah yeah. It runs actually for both Sequential and Local Executor :)